tasks/main.yml

```diff
@@ -834,8 +831,7 @@
         __mssql_sql_files_to_input:
           - ad_create_login.j2
         __mssql_ad_user: >-
-          {{ ad_integration_realm.split('.')
-          | first }}\{{ ad_integration_user }}
+          {{ ad_integration_realm }}\{{ ad_integration_user }}
```

Comment:
@JasonN3 CREATE LOGIN doesn't work with FQDN domain name:
```
CREATE LOGIN [domain-vh72.com\Administrator] FROM WINDOWS;
Windows NT user or group 'domain-vh72.com\Administrator' not found. Check the name again.
```
Works fine when providing domain name without TLD - `[domain-vh72\Administrator]`. Do you know how this must be provided in the case when domain consists of 2+ subnets?

Reply:
I am adding a new variable `mssql_ad_netbios_name` so that users can specify netbios name. The variable defaults to `"{{ ad_integration_realm.split('.') | first }}"`, i.e. the first subdomain of the value of ad_integration_realm.

Reply:
Did you try with `user@domain`?  Adding the variable `mssql_ad_netbios_name` would work and that's fine as a default value, but it would definitely be nicer if we can prompt for one less item.

Reply:
I have, this format doesn't work from sqlcmd
```
[root@client_mssql ~]# /opt/mssql-tools/bin/sqlcmd -U sa -P "p@55w0rD" -Q "CREATE LOGIN [Administrator@domain.cortoso.com] FROM WINDOWS;"
Msg 15407, Level 16, State 1, Server client_mssql, Line 1
'Administrator@domain.cortoso.com' is not a valid Windows NT name. Give the complete name: <domain\username>.
[root@client_mssql ~]# /opt/mssql-tools/bin/sqlcmd -U sa -P "p@55w0rD" -Q "CREATE LOGIN [Administrator@domain.cortoso] FROM WINDOWS;"
Msg 15407, Level 16, State 1, Server client_mssql, Line 1
'Administrator@domain.cortoso' is not a valid Windows NT name. Give the complete name: <domain\username>.
[root@client_mssql ~]# /opt/mssql-tools/bin/sqlcmd -U sa -P "p@55w0rD" -Q "CREATE LOGIN [Administrator@domain] FROM WINDOWS;"
Msg 15407, Level 16, State 1, Server client_mssql, Line 1
'Administrator@domain' is not a valid Windows NT name. Give the complete name: <domain\username>.
```

Reply:
Alright, then `mssql_ad_netbios_name` is required. Everything else looks good to me.

Reply:
> Alright, then `mssql_ad_netbios_name` is required. Everything else looks good to me.

@JasonN3 Thanks for your help!  Are you able to test this from github, or do you require publishing to Galaxy?